### ISSUE TITLE
Regression, contract `call` on non-existent smart contracts should return `0x`

### ISSUE THREAD

**Issue by AlfredoG87** [hashgraph/hedera-mirror-node#7428]
### Description

When doing a contract `call` request to a non-existent account `to` it should return `0x`.

right now is returning the following error:
```
{
    "_status": {
        "messages": [
            {
                "message": "INVALID_TRANSACTION",
                "detail": "",
                "data": ""
            }
        ]
    }
}
```

### Steps to reproduce

```
curl --location 'https://previewnet.mirrornode.hedera.com/api/v1/contracts/call' \
--header 'Content-Type: application/json' \
--data '{
    "from": "0x130ac2cfb5b5167c140a62ae3e9f5c305ad70316",
    "to": "0x5555555555555555555555555555555555555555",
    "data": "0x5c36b186",
    "gas": 400000,
    "value": null,
    "estimate": false
}'

```

### Hedera network

testnet, previewnet, other

### Version

v0.94.1

### Operating system

None

**Comment by AlfredoG87** [comment]
I realized that the regression is not on the response status `400`, but on the error message. the relay is the one that handles the response into an `0x` response, however based on the [PR](https://github.com/hashgraph/hedera-json-rpc-relay/pull/1990) worked and modified work. it seems like the message is no longer `FAIL_INVALID` but `INVALID_TRANSACTION` does that mean that `FAIL_INVALID` will no longer appear, or should we continue to handle it.

I think this ticket is no longer needed, since we are able to handle the response on the relay. But would like to know if we should continue to support handling of `FAIL_INVALID` and why the change?

**Comment by IvanKavaldzhiev** [comment]
Hey @AlfredoG87, there is a chance we introduced this behaviour with this [fix](https://github.com/hashgraph/hedera-mirror-node/pull/7203). Even though, the status is now changed, `FAIL_INVALID` might still appear, for example in the case of specific precompile execution failures. The relay should continue handle both `INVALID_TRANSACTION` and `FAIL_INVALID`.

In my opinion, whenever the relay receives one of these two failures, it should return a JSON-RPC response with `0x0` as a result. The mirror node REST API is not directly `JSON-RPC` compatible and it is expected that we might throw exceptions and behave like a regular REST API. It is the relay, which should wrap these errors and return the default empty result to the user.
